lib/pymedphys/docs/users/howto/gamma/from-dicom.ipynb

```diff
@@ -4,9 +4,9 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    Patient_Orientation = []
</pre><p>There is already a map like this in <code>pymedphys</code> :) See https://github.com/pymedphys/pymedphys/blob/fd91592173bed1220cf5feb170f865226301ad60/lib/pymedphys/_dicom/orientation.py#L65</p><p> </p><p>Also, I believe convention is to place "decubitis" in front of "left/right".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1770/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Nice! I did not know it.

Reply:
Also, I believe convention is to place "decubitis" in front of "left/right".
Yes, you are right. :-)